README.md

```diff
@@ -61,3 +61,492 @@ julia> logpdf_with_trans(dist, x, false) # ignoring the transformation
+
+julia> y = b(x)
+0.9312689879144197
+```
+
+In this case we see that `bijector(d::Distribution)` returns the corresponding constrained-to-unconstrained bijection for `Beta`, which indeed is a `Logit` with `a = 0.0` and `b = 1.0`. The resulting `Logit <: Bijector` has a method `(b::Logit)(x)` defined, allowing us to call it just like any other function. Comparing with the above example, `b(x) == link(dist, x)`. Just to convince ourselves:
+
+```julia
+julia> b(x) == link(dist, x)
+true
+```
+
+What about `invlink`?
+
+```julia
+julia> b⁻¹ = inv(b)
+Inversed{Logit{Float64}}(Logit{Float64}(0.0, 1.0))
+
+julia> b⁻¹(y)
+0.7173326646959575
+
+julia> b⁻¹(y) == invlink(dist, y)
+true
+```
+
+Pretty neat, huh? `Inversed{Logit}` is also a `Bijector` where we've defined `(ib::Inversed{<:Logit})(y)` as the inverse transformation of `(b::Logit)(x)`. Note that it's not always the case that `inv(b) isa Inversed`, e.g. the inverse of `Exp` is simply `Log` so `inv(Exp()) isa Log` is true. Also, we can _compose_ bijectors:
+
+```julia
+julia> id_y = (b ∘ b⁻¹)
+Composed{Tuple{Inversed{Logit{Float64}},Logit{Float64}}}((Inversed{Logit{Float64}}(Logit{Float64}(0.0, 1.0)), Logit{Float64}(0.0, 1.0)))
+
+julia> id_y(y) ≈ y
+true
+```
+
+And since `Composed isa Bijector`:
+
+```julia
+julia> id_x = inv(id_y)
+Composed{Tuple{Inversed{Bijectors.Logit{Float64}},Bijectors.Logit{Float64}}}((Inversed{Bijectors.Logit{Float64}}(Bijectors.Logit{Float64}(0.0, 1.0)), Bijectors.Logit{Float64}(0.0, 1.0)))
+
+julia> id_x(x) ≈ x
+true
+```
+
+This far we've seen that we can replicate the functionality provided by `link` and `invlink`. To replicate `logpdf_with_trans` we instead provide a `TransformedDistribution <: Distribution` implementing the `Distribution` interface from Distributions.jl:
+
+```julia
+julia> using Bijectors: TransformedDistribution
+
+julia> td = transformed(dist)
+TransformedDistribution{Beta{Float64},Logit{Float64},Univariate}(
+dist: Beta{Float64}(α=2.0, β=2.0)
+transform: Logit{Float64}(0.0, 1.0)
+)
+
+julia> td isa UnivariateDistribution
+true
+
+julia> logpdf(td, y)
+-1.0577727579778098
+
+julia> logpdf_with_trans(dist, x, true)
+-1.05777275797781
+```
+
+When computing `logpdf(td, y)` where `td` is the _transformed_ distribution corresponding to `Beta(2, 2)`, it makes more semantic sense to compute the pdf of the _transformed_ variable `y` rather than using the "un-transformed" variable `x` to do so, as we do in `logpdf_with_trans`. With that being said, we can also do
+
+```julia
+julia> logpdf_forward(td, x)
+-1.05777275797781
+```
+
+In the computation of both `logpdf` and `logpdf_forward` we need to compute `log(abs(det(jacobian(inv(b), y))))` and `log(abs(det(jacobian(b, x))))`, respectively. This computation is available using the `logabsdetjac` method
+
+```julia
+julia> logabsdetjac(b⁻¹, y)
+-1.595700144883034
+
+julia> logabsdetjac(b, x)
+1.595700144883034
+```
+
+Notice that
+
+```julia
+julia> logabsdetjac(b, x) ≈ - logabsdetjac(b⁻¹, y)
```

Comment:
Maybe `- logabsdetjac(b⁻¹, y)` -> `-logabsdetjac(b⁻¹, y)`, i.e. removing the space.